build.sbt

```diff
@@ -1,24 +1,28 @@
-import uk.gov.hmrc.DefaultBuildSettings.{defaultSettings, integrationTestSettings, scalaSettings}
+import uk.gov.hmrc.DefaultBuildSettings
+
+ThisBuild / majorVersion := 1
+ThisBuild / scalaVersion := "2.13.12"
 
 lazy val microservice = Project("email-verification", file("."))
   .enablePlugins(Seq(play.sbt.PlayScala, SbtDistributablesPlugin) *)
-  .settings(majorVersion := 1)
-  .settings(scalaSettings *)
-  .settings(defaultSettings() *)
+  .settings(DefaultBuildSettings.scalaSettings *)
+  .settings(DefaultBuildSettings.defaultSettings() *)
+  .settings(scalafmtOnCompile := true)
   .settings(ScoverageSettings())
-  .settings(ScalariformSettings())
-  .settings(scalaVersion := "2.13.8")
   .settings(scalacOptions ++= Seq(
     "-feature", "-deprecation",
     "-Werror",
-    "-Wconf:src=routes/.*&cat=unused-imports:silent"
+    "-Wconf:src=routes/.*&cat=unused-imports:silent",
+    "-Wconf:src=routes/.*:s"
   ))
   .settings(
-    libraryDependencies ++= AppDependencies(),
+    libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test,
```

Comment:
I think this is an anti-pattern as non of the fields are needed elsewhere, so it is better to hide inside the apply method